src/PolykeyAgent.ts

```diff
@@ -546,10 +546,6 @@ class PolykeyAgent {
       nodeId: this.keyManager.getNodeId(),
       clientHost: this.grpcServerClient.host,
       clientPort: this.grpcServerClient.port,
-      ingressHost: this.revProxy.ingressHost,
```

Comment:
We need `ingressHost` and `ingressPort` mainly so that it can be used later. But the proxy host and port doesn't.

Reply:
Actually dw about this. It's not that important. We can add this in later when we need it.